code/modules/client/preference_setup/occupation/occupation.dm

```diff
@@ -126,7 +126,7 @@
 		if(alt_title && !(alt_title in job.alt_titles))
 			pref.player_alt_titles -= job.title
 
-/datum/category_item/player_setup_item/occupation/content(mob/user, limit = 16, list/splitJobs = list("Chief Medical Officer"))
+/datum/category_item/player_setup_item/occupation/content(mob/user, limit = 16, list/splitJobs = list("Chief Engineer"))
```

Comment:
That is used to split the job list in a new column on the crew manifest.

Reply:
The atmospheric technician role took up an entire column because it split at Chief Medical Officer; the addition of the Journalist role (after adding the Merchant, I believe) was enough to cause this. I just moved the split to the Chief Engineer spot.
This is for table formatting.